### PR TITLE
Update npm install for publish job

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Update install step on publish job

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -224,10 +224,10 @@ jobs:
       - checkout
       - restore_cache: *restore_cache
       - run: *gh-config
-      - run: *install
```
Comment (lcampos) [inline .circleci/config.yml:227]: Having this as an alias is causing install issues so I'm updating it to a normal `npm install` in the job.

File: .circleci/config.yml [hunk 2]
```diff
@@ -224,10 +224,10 @@ jobs:
...
       - run:
           name: 'Install build dependencies'
           command: |
+            npm install
```
Comment (brpowell) [inline .circleci/config.yml:230]:
```suggestion
            npm install --only=prod
```
Is this needed, or no?

Comment (AnanyaJha) [reply .circleci/config.yml:230]: what does the `--only=prod` do?

Comment (lcampos) [reply .circleci/config.yml:230]: Not needed, setting node_env to production in the compile step will take care of what we need

Comment (brpowell) [reply .circleci/config.yml:230]: @AnanyaJha It avoids installing the dev dependencies for a production build

**Pull request merged**